pom.xml

```diff
@@ -38,4 +46,27 @@
       </plugin>
     </plugins>
   </build>
-</project>
\ No newline at end of file
+  
+  <profiles>
```

Comment:
Follow google-cloud-java's root pom.xml https://github.com/googleapis/google-cloud-java/blob/main/pom.xml#L176
It seems following it already.

Reply:
Made it identical with google-cloud-java's root pom.xml